README.md

```diff
@@ -1 +1,41 @@
-# MeasurementConv
\ No newline at end of file
+# Measurement Conversion Library
+
+This library will make it easy to add/combine ingredients of 
+different sizes and get the computed result in a format 
+that is the most convenient.
+
```

Comment:
I will try my best to make my program work the way that you’ve described :)

Reply:
We'll both work on it. You'll start things off, then I'll build on your work.